functions/shared_libs/api_clients/zoom_client.py

```
@@ -56,78 +120,509 @@ def fetch_access_token(self):
+            return self.aggregate_participants(
+                [dict(p, _meeting_uuid=None, _instance_start=None) for p in participants]
+            )
+
+        all_rows: List[Dict[str, Any]] = []
+        for inst in relevant:
+            uuid = inst.get("uuid")
+            if not uuid:
+                continue
+            try:
+                participants = self.get_meeting_instance_participants(uuid)
+            except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)